backend/src/main/java/gov/cdc/usds/simplereport/api/model/UserPermission.java

```diff
@@ -0,0 +1,11 @@
+package gov.cdc.usds.simplereport.api.model;
+
+public enum UserPermission {
+	READ_PATIENT_LIST,
+	READ_RESULT_LIST,
+	EDIT_PATIENT,
+	EDIT_FACILITY,
+	EDIT_ORGANIZATION,
+	START_TEST,
+	UPDATE_TEST;
```

Comment:
Can we add a `SUBMIT_TEST`, which should, at least for now, be assigned to the same roles as `START_TEST` and `UPDATE_TEST`

Here's a slack thread mildly requesting support for more granularity:
https://usds.slack.com/archives/G01KW3DGG0Y/p1611599907023600

Reply:
Can do.